### PR TITLE
🌐 Update German translation for `docs/features.md`

### PR DESCRIPTION
**Pull request by jomue**
I've fixed a few typos in the german translation.

### CODE REVIEW

File: docs/de/docs/features.md [hunk 1]
```diff
@@ -142,8 +142,8 @@ FastAPI enthält ein extrem einfaches, aber extrem mächtiges <abbr title='oft v
 * **Automatische Umsetzung** durch FastAPI.
 * Alle abhängigen Komponenten könnten Daten von Anfragen, **Erweiterungen der Pfadoperations-**Einschränkungen und der automatisierten Dokumentation benötigen.
 * **Automatische Validierung** selbst für *Pfadoperationen*-Parameter, die in den Abhängigkeiten definiert wurden.
-* Unterstütz komplexe Benutzerauthentifizierungssysteme, mit **Datenbankverbindungen**, usw.
-* **Keine Kompromisse** bei Datenbanken, Eingabemasken, usw. Sondern einfache Integration von allen.
+* Unterstützt komplexe Benutzerauthentifizierungssysteme, mit **Datenbankverbindungen**, usw.
+* **Keine Kompromisse** bei Datenbanken, Eingabemasken, usw. sondern einfache Integration von allen.
```
Comment (mawassk) [inline docs/de/docs/features.md:146]: * **Keine Kompromisse** bei Datenbanken, Eingabemasken, usw., sondern einfache Integration von allen.

**Review by mawassk — CHANGES_REQUESTED** [review]

File: docs/de/docs/features.md (hunk 1 above)
Comment (jomue) [reply docs/de/docs/features.md]: Thanks 👍

**Review by mawassk — APPROVED** [review]

**Comment by ArcLightSlavik** [comment]
Could you update the name of PR to `Update German Translation for docs/features.md`

**Review by rogerbrinkmann — APPROVED** [review]
Proposed changes are necessary and correct

**Review by jsbeckr — APPROVED** [review]
fixes typos and mistranslation 👍

**Comment by tiangolo** [comment]
Awesome, thank you @jomue! :bow: 

And thanks for the reviews @jsbeckr, @rogerbrinkmann, @mawassk! :cake: 

And thanks for the help @ArcLightSlavik! :sunglasses:

**Pull request merged**